slack_global_report.py

```diff
@@ -0,0 +1,241 @@
+#! /usr/bin/env python
+
+import copy
+import json
+import random
+import time
+import sys
+
+import slack
+
+import random_name
+import config
+import channel
+import channel_members_log
+import user
+import utils
+import enricher
+import slack_token
+import slack_formatter
+
+
+class SlackGlobalReport(object):
+
+    def __init__(self, fake=False):
+        self.sf = slack_formatter.SlackFormatter()
+        random.seed(time.time())
```

Comment:
What is this for?

Reply:
Some old code.  Might need to resurrect at some point -- goal is to be able to basically create fake activity reports so we can show people what this does without violating privacy, but I'm not going to worry about it for now.  Clean it up.